modulegen/internal/mkdocs/main.go

```diff
@@ -26,8 +26,8 @@ func (g Generator) AddModule(ctx context.Context, tcModule context.Testcontainer
 	if err != nil {
 		return fmt.Errorf("generate md file: %w", err)
 	}
-	moduleMd := tcModule.ParentDir() + "/" + tcModule.Lower() + ".md"
-	indexMd := tcModule.ParentDir() + "/index.md"
+	moduleMd := filepath.Join(tcModule.ParentDir(), tcModule.Lower()) + ".md"
```

Comment:
suggestion: add the extension inside the join, for consistency with other.s
```suggestion
	moduleMd := filepath.Join(tcModule.ParentDir(), tcModule.Lower() + ".md")
```